### PR TITLE
New package: Dreamacro.clash version 1.17.0

### PR DESCRIPTION
**Pull request by CoolPlayLin**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/115858&drop=dogfoodAlpha

### CODE REVIEW

**Comment by CoolPlayLin** [comment]
.

**Comment by stephengillie** [comment]
>  Manifest Error: Relative file path must not point to a location outside of archive directory. [RelativeFilePath]

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2023-08-16 10:52:45.625 [CLI ] Path points to a location outside of the install directory: C:/clash-windows-386.exe

File: manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+PackageIdentifier: Dreamacro.clash
+PackageVersion: 1.17.0
+InstallerType: zip
+NestedInstallerType: portable
+UpgradeBehavior: uninstallPrevious
+ReleaseDate: 2023-06-30
+Installers:
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: /clash-windows-386.exe
```
Comment (stephengillie) [inline manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml:13]:
```suggestion
  - RelativeFilePath: ./clash-windows-386.exe
```

File: manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    PortableCommandAlias: clash-windows-386
+  InstallerUrl: https://github.com/Dreamacro/clash/releases/download/v1.17.0/clash-windows-386-v1.17.0.zip
+  InstallerSha256: F2494A8A29EF47E5A66A6A7C32B2DDD37034E91857FDC68B6387B077AF4A514B
+- Architecture: x64
+  NestedInstallerFiles:
+  - RelativeFilePath: /clash-windows-amd64.exe
```
Comment (stephengillie) [inline manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml:19]:
```suggestion
  - RelativeFilePath: ./clash-windows-386.exe
```

File: manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    PortableCommandAlias: clash-windows-amd64
+  InstallerUrl: https://github.com/Dreamacro/clash/releases/download/v1.17.0/clash-windows-amd64-v1.17.0.zip
+  InstallerSha256: 81C6FDF813B27065F37B151DAA7FD475178057A3981C5D5D191FD387C792F4BB
+- Architecture: arm
+  NestedInstallerFiles:
+  - RelativeFilePath: /clash-windows-armv7.exe
```
Comment (stephengillie) [inline manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml:25]:
```suggestion
  - RelativeFilePath: ./clash-windows-armv7.exe
```

File: manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    PortableCommandAlias: clash-windows-armv7
+  InstallerUrl: https://github.com/Dreamacro/clash/releases/download/v1.17.0/clash-windows-armv7-v1.17.0.zip
+  InstallerSha256: A8746B42C5C4131D5555BC958BFBE8FBC228D9E70CF2D87428B2AE77FC4AB2D8
+- Architecture: arm64
+  NestedInstallerFiles:
+  - RelativeFilePath: /clash-windows-arm64.exe
```
Comment (stephengillie) [inline manifests/d/Dreamacro/clash/1.17.0/Dreamacro.clash.installer.yaml:31]:
```suggestion
  - RelativeFilePath: ./clash-windows-arm64.exe
```

**Review by stephengillie — APPROVED** [review]

**Pull request merged**